README.md

```diff
@@ -20,6 +20,315 @@ env MIGRATION_DB_NAME=stay-informed-test npm run migrate
 
 And `npm test` should work at this point
 
+## Configuring API Keys
+
+Stay Informed leverages the [Open Secrets API](https://opensecrets.org/open-data/api-documentation), [News API](https://www.npmjs.com/package/newsapi), [Google Civics API](https://developers.google.com/civic-information/), and [ProPublica Congress API](https://www.propublica.org/datastore/api/propublica-congress-api)
+
+Sign up for API keys and save them to your `.env` file using the following variable names so they can be referenced in the `config.js` and other services:
+
+#### Civics API
+
+* `CIVIC_API_KEY`
+* `CIVIC_API_URL="https://www.googleapis.com/civicinfo/v2/representatives"`
+
+#### ProPublica
+
+* `PROPUBLICA_API_KEY`
+
+#### News API
+
+* `NEWS_API_KEY`
+
+#### Open Secrets
+
+* `OPEN_SECRETS_API_KEY`
+
+
+## API Responses
+
+#### Civics API
+
+`GET https://www.googleapis.com/civicinfo/v2/representatives??key=${config.CIVIC_API_KEY}&address=${address}`
+
+This returns a user's state, congressional district, and other data associated with this registered address.
+
+For our needs, we take the state and congressional district data and use them in our request to ProPublica and Open Secrets
+
+```
+{
+ "kind": "civicinfo#representativeInfoResponse",
+ "normalizedInput": {
+  "line1": "227 East 2nd Street",
+  "city": "New York",
+  "state": "NY",
+  "zip": "10009"
+ },
+ "divisions": {
+  "ocd-division/country:us": {
+   "name": "United States",
+   "officeIndices": [
+    0,
+    1
+   ]
+  },
+  "ocd-division/country:us/state:ny": {
+   "name": "New York",
+   "officeIndices": [
+    2,
+    4,
+    5,
+    14,
+    15
+   ]
+  },
+  "ocd-division/country:us/state:ny/cd:12": {
+   "name": "New York's 12th congressional district",
+   "officeIndices": [
+    3
+   ]
+  },
+  "ocd-division/country:us/state:ny/county:new_york": {
+   "name": "New York County",
+   "alsoKnownAs": [
+    "ocd-division/country:us/state:ny/borough:manhattan",
+    "ocd-division/country:us/state:ny/place:new_york/county:new_york"
+   ],
+   "officeIndices": [
+    11,
+    12
```

Comment:
I think the spacing/indentation is off here

Reply:
Was just a copy/paste from the response in Postman